### ISSUE TITLE
Embedding YouTube videos in posts

### ISSUE THREAD

**Issue by manzanotti** [poppastring/dasblog-core#549]
I've embedded a youtube video in a post, and I get the following:

![image](https://user-images.githubusercontent.com/2589151/112749979-bd276a80-8fbd-11eb-905d-8e8301fe65b2.png)

I've since added www.youtube.com to both the style and script sources on the admin page, but that hasn't made a difference.

Any idea what I'm doing wrong here?

**Comment by poppastring** [comment]
@manzanotti Usually when I am dealing with YouTube videos I right click on the video and copy embed code as follows:
![image](https://user-images.githubusercontent.com/8739371/113079718-962a8d80-91a3-11eb-9f9c-2ca3ecb70096.png)

I open OLW and I usually go straight to source:
![image](https://user-images.githubusercontent.com/8739371/113080194-80699800-91a4-11eb-807a-ec0651ac793a.png)

And insert the code I copied where I need, this is what the code looks like:
`<iframe width="1536" height="754" src="https://www.youtube.com/embed/AbSehcT19u0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`

I noticed that your blog post has some interesting additional divs this for example:
`<div class="icon icon-blocked" jseval="updateIconClass(this.classList, iconClass)" jstcache="2"></div>`  

I would be begin by removing everything that is not specifically inside the <iframe>.

**Comment by manzanotti** [comment]
@poppastring Apologies, I've got to stop leaving info out of my issues!

I had looked at the code that YouTube provide for embedding the video, but I did the classic developer trick of speed reading it, and was adamant that it was the same as what OLW was inserting.

It turns out that OLW is, for reasons I can't fathom, changing the source url protocol from the https in the link I gave it, to http. And that is what is causing the issue! I'll go raise an issue on the OLW repo, and see if I can fix it.

FYI, that extra html is inserted by either YouTube or the browser.
